python/lsst/sims/GalSimInterface/galSimPSF.py

```diff
@@ -155,7 +156,49 @@ def __init__(self, fwhm=0.6):
         #for r at half the maximum of the PSF
         alpha = fwhm/2.3835
 
-        gaussian1 = galsim.Gaussian(sigma=alpha)
-        gaussian2 = galsim.Gaussian(sigma=2.0*alpha)
+        rms_pixel_size = pixel_scale*pixel_scale/6.0
```

Comment:
I suspect he was thinking that T = Ixx + Iyy = scale^2/6 for a pixel.  I'm pretty sure he just forgot that the same quantity for a Gaussian is 2sigma^2, not just sigma^2.